### ISSUE TITLE
A way to order tests

### ISSUE THREAD

**Issue by jim4067** [foundry-rs/foundry#4724]
### Component

Forge

### Describe the feature you would like

Can you select the order in which tests get run? 

In my case, they are very random. It would be nice to have them run in a specified order, maybe from the first one to the last one or the last one to the first one.

**Comment by mds1** [comment]
Tests are run in parallel, which is why the order appears random—results are shown in the order they complete. What is the reason you need a particular order? Things like `forge snapshot` should always display them in the same order
